src/tactic/obviously.lean

```diff
@@ -1,5 +1,10 @@
+/-
+Copyright (c) 2017 Scott Morrison. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: Scott Morrison
+-/
 import tactic.tidy
-import tactic.basic
+import tactic.replacer
 
```

Comment:
How about a module doc, while you're at it? :)

Reply:
Good call!